src/main/java/hudson/remoting/RemoteClassLoader.java

```diff
@@ -105,7 +105,14 @@
      */
     private final Map<String,ClassReference> prefetchedClasses = Collections.synchronizedMap(new HashMap<String,ClassReference>());
 
-    public static ClassLoader create(ClassLoader parent, IClassLoader proxy) {
+    /**
+     * Creates a remotable classloader
+     * @param parent Parent classloader. Can be {@code null} if there is no delegating classloader
```

Comment:
what does `null` mean... does it mean that the parent is supposed to be the receiving channel's base classloader?